docs/r2dbc.md

```diff
@@ -30,7 +30,7 @@ compile 'com.google.cloud.sql:cloud-sql-connector-r2dbc-mysql:1.15.1'
 
 **Note:** Also include the R2DBC Driver for
 MySQL, `io.asyncer:r2dbc-mysql:<LATEST-VERSION>`
-<!-- {x-release-please-end} -->
+<!-- {x-version-update-end} -->
```

Comment:
r2dbc README is currently working as expected, is this just a naming change?

Reply:
Because it started with `<!-- {x-version-update-start-..} ` so it should finish with `<!-- {x-version-update-end} -->`. It will continue to work as expected.